src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
+    {
+        $dir   = dirname($origFile);
+        // matches any type of import rule
+        preg_match_all('~@import\s*(?:url)?(?:\(?\'?\"?)?([^\'\"\)\(]*)(?:\'?\"?)?\)?\s?([^;]*);~im',$content,$imports,PREG_SET_ORDER);
+        if(count($imports))
```

Comment:
That's also a habit. I always check double. In fact, normally I would do it like so:

    if(is_array($imports) && count($imports)) {